eth2/utils/ssz/fuzz/fuzz_targets/fuzz_target_vec_decode.rs

```diff
@@ -0,0 +1,21 @@
+#![no_main]
+#[macro_use] extern crate libfuzzer_sys;
+extern crate ethereum_types;
+extern crate ssz;
+
+use ethereum_types::{Address, H256};
+use ssz::{DecodeError, Decodable};
+
+// Fuzz ssz_decode()
+fuzz_target!(|data: &[u8]| {
+    let _result: Result<(Vec<u8>, usize), DecodeError> = Decodable::ssz_decode(data, 0);
+    /*
```

Comment:
Are these all covered in by the other `fuzz_target_<type>.rs` files? If so, can we strip out these comments?

Reply:
So all the other types are covered by other tests but Vec<Types> are not covered.

The reason I originally had them there is cause they have different ssz size for the types and I think it's worth having a few tests for different sizes but it's also bad practice to be testing multiple different things in one fuzz target hence I commented them out.

To sum up we should probably have a few different fuzz targets for different vec types rather than just u8 but they should be in their own files. (I'll do this now)